Calibration/PPSAlCaRecoProducer/test/test_prompt_AlCaRecoProducer.sh

```diff
@@ -0,0 +1,12 @@
+# I've also tried simpler tests without L1Reco stage, namely:
+cmsDriver.py testExpressPPSAlCaRecoProducer  -s ALCAPRODUCER:PPSCalMaxTracks,ENDJOB \
+--process ALCARECO \
+--scenario pp \
+--era ctpps_2018 \
+--conditions auto:run3_data_express \
```

Comment:
```suggestion
--conditions auto:run3_data_prompt \
```

Reply:
If I remember correctly there was an issue in missing LHCInfo records in prompt GT, therefore we've been using express GT. I will check if this is still the case

Reply:
Indeed, the problem is related to the missing LHCInfo record. If I switch to `auto:run3_data_prompt` then I get the following error when running a test on a fake ALCARAW data.  
```
Begin processing the 1st record. Run 322022, Event 1760583922, LumiSection 1025 on stream 0 at 02-Feb-2022 21:52:06.155 CET
----- Begin Fatal Exception 02-Feb-2022 21:52:07 CET-----------------------
An exception of category 'NoRecord' occurred while
   [0] Processing  Event run: 322022 lumi: 1025 event: 1760583922 stream: 0
   [1] Running path 'ALCARECOoutput_step'
   [2] Prefetching for module PoolOutputModule/'ALCARECOoutput'
   [3] Prefetching for module CTPPSProtonProducer/'ctppsProtons'
   [4] Calling method for EventSetup module CTPPSInterpolatedOpticalFunctionsESSource/'ctppsInterpolatedOpticalFunctionsESSource'
   [5] While getting dependent Record from Record CTPPSInterpolatedOpticsRcd
Exception Message:
No "LHCInfoRcd" record found in the EventSetup.

 The Record is delivered by an ESSource or ESProducer but there is no valid IOV for the synchronization value.
 Please check 
   a) if the synchronization value is reasonable and report to the hypernews if it is not.
   b) else check that all ESSources have been properly configured. 
----- End Fatal Exception -------------------------------------------------
```

Reply:
I can add some comment to the script as well

Reply:
I added some comments in https://github.com/cms-sw/cmssw/pull/36273/commits/18c83a8f88a9ba2168b0438515eb0d211ecdd53c